app-libs/stf/src/stf_sgx.rs

```diff
@@ -115,6 +118,26 @@ impl Stf {
 					} else {
 						None
 					},
+				#[cfg(feature = "evm")]
+				TrustedGetter::evm_account_codes(_who, evm_account) =>
+				// TODO: This probably needs some security check if who == evm_account (or assosciated)
```

Comment:
Can you add an issue for this?

I already have an idea how to do this, but probably makes sense to do this in a separate issue.

Reply:
https://github.com/integritee-network/worker/issues/935